StripeIdentity/StripeIdentity/Source/API Bindings/IdentityAPIClient.swift

```diff
@@ -88,7 +88,7 @@ final class IdentityAPIClientImpl: IdentityAPIClient {
     func getIdentityVerificationPage() -> Promise<StripeAPI.VerificationPage> {
         return apiClient.get(
             resource: APIEndpointVerificationPage(id: verificationSessionId),
-            parameters: [:]
+            parameters: ["app_identifier": Bundle.main.bundleIdentifier ?? ""]
```

Comment:
Why not just call this "bundle_identifier" ?

Reply:
this is a cross platform parameter, it's also called this on Android, and "bundle" is not a keyword on Android